server/routes/utilMongoose.js

```diff
@@ -323,53 +323,28 @@ router.post('/addAlumni/', async (req, res, next) => {
         const gradYear = parseInt(req.body.gradYear);
         const location = req.body.location;
         const profession = req.body.profession;
-        const company = req.body.company;
-        const college = req.body.college;
+        const country = req.body.country;
+        const city = req.body.city;
         const availabilities = [];
-        const timeZone = req.body.timeZone;
         const zoomLink = req.body.zoomLink;
         const password = req.body.password;
```

Comment:
instead of re-creating logic to add user and alumni objects, utilise `createAlumni` helper method already in use by `/util/seed`.